ssc/cmod_pvsamv1.cpp

```diff
@@ -1524,6 +1539,51 @@ void cm_pvsamv1::exec( ) throw (compute_module::general_error)
 	// The shared inverter of the PV array and a tightly-coupled DC connected battery
 	std::unique_ptr<SharedInverter> sharedInverter(new SharedInverter(inv_type, num_inverters, &snlinv, &plinv));
 
+	// Inverter thermal derate curves
+	bool en_inv_tdc = as_boolean("en_inv_tdc");
+	if (en_inv_tdc) {
+		double* curve1 = new double[3];
+		curve1[0] = as_double("inv_tdc_V1");
+		curve1[1] = as_double("inv_tdc_T1");
+		curve1[2] = as_double("inv_tdc_S1");
+		
+		double* curve2 = new double[3];
+		try {
```

Comment:
Try/catch is inconsistent with how we;ve typically error-checked things in the past, not necessarily a bad thing but I don't know enough about those functions. Will they build across all platforms for the right C++ standard (11)? @nickdiorio  your input on this is welcome

Reply:
try catch is a standard phrase and is used in ssc/core.cpp for instance. It was implemented in case there was no value assigned to some of these variables (a SDK user who only needed one curve so did not bother to assign the rest)

But I changed to use a matrix and that simplified this section